packages/compass-context-menu/src/context-menu-provider.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { render } from '@mongodb-js/testing-library-compass';
+import { expect } from 'chai';
+import { ContextMenuProvider } from './context-menu-provider';
+import type { ContextMenuWrapperProps } from './types';
+
+describe('ContextMenuProvider', function () {
```

Comment:
this isn't related to the failing test but was just part of the changes that did not end up included in the branch I was intending to merge